content/docs/user-guide/engine/_index.md

```diff
@@ -1,14 +1,34 @@
 ---
-description: The following topics will guide you through programming with and for O3DE. 
+description: Learn about Open 3D Engine's core systems and how you can extend the systems by programming.
 title: Open 3D Engine Core Programming
 linktitle: Engine Core
 weight: 600
 ---
 
-## Topics
+The *engine core* encompasses the source code, systems, and modules that underly **Open 3D Engine (O3DE)** and its tools. All of these tools -- Editor, Asset Processor, Script Canvas, and so on -- work together to build an interactive 3D experience that uses modern rendering techniques and game play logic, and move your design from an empty scene to a fully polished project that you can deploy and share. In this section, you'll gain a deeper understanding in each part of the engine core and learn how you can expand O3DE's toolkit.
 
-| Topic Area | Description |
-| --- | --- |
-| [Event Bus](./ebus) (EBus) | Learn about EBus, the messaging infrastructure for notifications and messaging. |
-| [AzAutoGen Generator](./autogen) | Learn about the `AzAutoGen` tool, used to create code or resources from [Jinja2 templates](https://jinja.palletsprojects.com/en/3.0.x/) and XML definition files. |
-| [Memory management](./memory) | Learn how O3DE allocates and manages memory resources for games, as well as memory debugging and overrun detection. |
+Start by learning how to navigate the O3DE directory structure and familiarize yourself with the core modules in the O3DE SDK (refer to [Overview of O3DE SDK](/docs/welcome-guide/key-concepts.md) in the Key Concepts). These modules are the entry points to all of the functionality in O3DE. As you proceed throughout this section, use an IDE of your choice to view the source code and refer to the [API Reference](/docs/api/) in the O3DE Documentation.
```

Comment:
```suggestion
Start by learning how to navigate the O3DE directory structure and familiarize yourself with the core modules in the O3DE SDK. For details, refer to [Overview of the O3DE SDK](/docs/welcome-guide/key-concepts/#overview-of-the-o3de-sdk) in the Key Concepts topic. The core modules are the entry points to all of the functionality in O3DE. As you proceed through this section, use an IDE of your choice to view the source code, and refer to the [API Reference](/docs/api/) in the O3DE Documentation.
```
* I recommend moving the parenthetical cross-reference link into its own sentence. IMO, that looks much cleaner than putting it in parentheses at the end of a long sentence.
* The cross-reference link text refers to a section, but the link doesn't include the anchor. Revise to use the anchor link. (Note that I haven't tested it as revised.)
* Consider adding a comma after "view the source code" so that it doesn't read as "use an IDE of your choice to...refer to the API Reference...."